src/python/grpcio/grpc/_server.py

```diff
@@ -828,7 +828,9 @@ def stop(self, grace):
         return _stop(self._state, grace)
 
     def __del__(self):
-        _stop(self._state, None)
+        if hasattr(self, "_state"):
```

Comment:
style nit: use single quotes for strings to be consistent throughout the file (yapf does not catch this?)

Reply:
YAPF seems fine with double quotes